apis/Google.Devtools.Aspnet/Google.Devtools.Aspnet.Tests/Google.Devtools.Aspnet.Tests.csproj

```diff
@@ -0,0 +1,73 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is just a class library, right, even though it targets the main framework? If so, it would be more consistent if you could use project.json for that. Happy to talk this over with you. This would also remove the packages.config file.